test/e2e/kubectl/kubectl.go

```diff
@@ -2053,10 +2053,10 @@ func modifyReplicationControllerConfiguration(contents string) io.Reader {
 	return bytes.NewReader(data)
 }
 
-func forEachReplicationController(c clientset.Interface, ns, selectorKey, selectorValue string, fn func(v1.ReplicationController)) {
+func forEachReplicationController(c clientset.Interface, ns, selectorKey, selectorValue string, podListTimeout time.Duration, fn func(v1.ReplicationController)) {
 	var rcs *v1.ReplicationControllerList
 	var err error
-	for t := time.Now(); time.Since(t) < framework.PodListTimeout; time.Sleep(framework.Poll) {
+	for t := time.Now(); time.Since(t) < podListTimeout; time.Sleep(framework.Poll) {
```

Comment:
why passed as an argument instead of using the timeout?
we are using `framework.Poll` either way

Reply:
ah, nevermind, this is part of the TimeoutContext now